packages/next/src/next-devtools/dev-overlay/components/overview/segment-explorer.tsx

```diff
@@ -9,13 +9,23 @@ const isFileNode = (node: SegmentTrieNode) => {
   return !!node.value?.type && !!node.value?.pagePath
 }
 
-function PageSegmentTree({ tree }: { tree: SegmentTrieNode }) {
+function PageSegmentTree({
+  tree,
+  isAppRouter,
+}: {
+  tree: SegmentTrieNode
+  isAppRouter: boolean
+}) {
   return (
     <div
       className="segment-explorer-content"
       data-nextjs-devtool-segment-explorer
     >
-      <PageSegmentTreeLayerPresentation node={tree} level={0} segment="" />
+      {isAppRouter ? (
+        <PageSegmentTreeLayerPresentation node={tree} level={0} segment="" />
+      ) : (
+        <p>Route Info currently is only available for the App Router.</p>
```

Comment:
Addressed the comment to add a note for availability in pages router https://github.com/vercel/next.js/pull/80668#pullrequestreview-2942419062